core/src/main/scala/chisel3/BitsImpl.scala

```diff
@@ -99,7 +99,7 @@ private[chisel3] trait BitsImpl extends Element { self: Bits =>
   }
 
   protected def _applyImpl(x: UInt)(implicit sourceInfo: SourceInfo): Bool =
-    do_extract(x)
+    extract(x)
```

Comment:
It's probably fine, but I think all of these should be calling the `_*impl` versions